dll/directx/wine/ddraw/utils.c

```diff
@@ -580,8 +580,17 @@ unsigned int wined3dmapflags_from_ddrawmapflags(unsigned int flags)
         wined3d_flags |= WINED3D_MAP_NO_DIRTY_UPDATE;
     flags &= ~(handled | DDLOCK_WAIT | DDLOCK_READONLY | DDLOCK_NODIRTYUPDATE);
 
+#ifndef __REACTOS__
     if (flags)
         FIXME("Unhandled flags %#x.\n", flags);
+#elif defined(__REACTOS__) && defined(DBG)
```

Comment:
I'm not sure why `defined(DBG)` matters.

Reply:
the goal of that is to prevent the creation of the static variable in rls-configuration.
I don't want to harm performance of release-build when speeding up the dbg-build

Reply:
superseded